cli/formatter.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
```

Comment:
Rename this file fmt.rs ?

Maybe give some //! explanation about how this works with links to dprint?